app/bundles/LeadBundle/Views/Lead/lead.html.php

```diff
@@ -205,7 +205,11 @@
                                                         <?php if (is_array($field['value']) && 'multiselect' === $field['type']): ?>
                                                             <?php echo implode(', ', $field['value']); ?>
                                                         <?php else: ?>
-                                                            <?php echo $field['value']; ?>
+                                                            <?php if ('select' === $field['type']): ?>
```

Comment:
For readability and consistency, can you change the order of this to variable first then string? 
i.e. $field['type'] == 'select'